### DISCUSSION TITLE
Is .Net 9 going to support native AOT on WinForms ?

### DISCUSSION THREAD

**Discussion by iamthefaker** [dotnet/aspnetcore#55318] [Q&A]
Is .Net 9 going to support native AOT on WinForms ?

**Comment by martincostello** [comment]
This isn't a question about ASP.NET Core - you should ask in [dotnet/winforms](https://github.com/dotnet/winforms).

**Reply by iamthefaker** [reply]
Ah, ok, sorry. I would delete it if I could figure out how to.

**Reply by gfoidl** [reply]
For reference -> https://github.com/dotnet/winforms/discussions/11257

So I'm closing this one here.
